src/main/java/sdmExam/Game.java

```diff
@@ -50,6 +50,8 @@ public void makeMove(Stone color, Position position) throws Exception {
 
         board.addStoneAt(color, position);
         lastPlay = color;
+
+        searchAndFillTerritories();
```

Comment:
Maybe we should think about this later. My idea is to make a separate method called play() in which you loop and make the moves.

Reply:
searchAndFillTerritories method moved into Board class.